content/blog/2021/08/DeployingCamelRouteInAWSLambda/index.md

```diff
@@ -0,0 +1,210 @@
+---
+title: "Deploying a Camel Route in AWS Lambda using Quarkus"
+date: 2021-08-19
+authors: [ravishankarhassain]
+categories: ["AWS", "Lambda","Camel Quarkus"]
+preview: "A new demonstration explaining how to develop a camel route using Quarkus (in both JVM & Native mode) and steps to deploy it in AWS Lamda"
+---
+
+Do you fancy running camel route as functions in AWS Lambda. Well I did a small Proof Of Concept to test this and the results were interesting.
+Thanks to the Quarkus & Camel-Quarkus project and the team for their efforts to make this technically possible.
```

Comment:
Why not "Thanks to the Quarkus **and** Camel-Quarkus **communities** for their efforts to make this technically possible".

As a side note, I'm sure this acknowledgment will be appreciated by contributors :)